contrib/inventory/ec2.py

```diff
@@ -399,7 +399,10 @@ def read_settings(self):
         if self.boto_profile:
             cache_dir = os.path.join(cache_dir, 'profile_' + self.boto_profile)
         if not os.path.exists(cache_dir):
-            os.makedirs(cache_dir)
+            try:
+                os.makedirs(cache_dir)
+            except(OSError):
+                print('Cache directory already exists. Skipping the step...')
```

Comment:
I don't think printing an informational message is that useful. Just `pass` here instead